tests/openwisp2/settings.py

```diff
@@ -135,17 +135,6 @@
     }
 ]
 
-if not TESTING:
```

Comment:
why?

Reply:
This code block was duplicated in the settings.py